sunpy/net/fido_factory.py

```diff
@@ -504,9 +504,9 @@ def _repr_html_(self):
         """
         This enables the "pretty" printing of the Fido Clients with html.
         """
-        return self._print_clients(visible_entries=15, html=True)
+        return self._print_clients(html=True)
 
-    def _print_clients(self, html=False, visible_entries=None):
+    def _print_clients(self, html=False, visible_entries=-1):
```

Comment:
Does `-1` do something else to `None` when passed into `pformat_all`?

Reply:
- If a negative value of max_lines is supplied then there is no line limit applied.
- If no value of max_lines is supplied then the height of the screen terminal is used to set max_lines.

Reply:
Interesting! Thanks for clarifying that.